src/components/pages/CityDetail/panes/HousingPane.js

```diff
@@ -1,5 +1,38 @@
 import React from "react";
-
+import { Tabs } from "antd";
 export default function HousingPane({ rent }) {
-  return <>Rental Prices: ${rent.studio}</>;
+  console.log(rent);
+  const { TabPane } = Tabs;
+  // This list will be the titles of the tabs
+  // it also matches the keys in rent (although they're all lowercase in rent)
+  const aptTypes = ["Studio", "1BR", "2BR", "3BR", "4BR"];
+
+  // Make a color-coded price change display
```

Comment:
Thanks for the comment, I wouldn't have known what this does--this looks super cool!